activiti-cloud-starter-audit/src/test/java/org/activiti/cloud/starter/audit/tests/it/AuditServiceIT.java

```diff
@@ -661,6 +733,49 @@ public void eventsShouldHaveMessageIdSet() {
 
         });
     }
+    
+    @Test
+    public void shouldGetSignalReceivedEvent() {
+        //given
+        List<CloudRuntimeEvent> coveredEvents = new ArrayList<>();
+        
+        BPMNSignalImpl signal = new BPMNSignalImpl("signalId");
+        signal.setProcessDefinitionId("processDefinitionId");
+        signal.setProcessInstanceId("processInstanceId");
+
+        SignalPayload signalPayload = ProcessPayloadBuilder.signal()
+                .withName("SignalName")
```

Comment:
Could you add variables to the signal payload as well?

Reply:
Done